src/storemodel.cpp

```
@@ -282,55 +282,106 @@ auto StoreModel::dropMimeData(const QMimeData *data, Qt::DropAction action,
     return true;
   }
+
+  dragAndDropInfoPasswordStore info;
+  if (!parseDropData(data, &info)) {
+    return false;
+  }
+
+  return executeDropAction(info, action, parent);
+}
+
+bool StoreModel::parseDropData(const QMimeData *data,
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)